packages/kbn-test/src/jest/utils/get_url.ts

```diff
@@ -25,6 +25,7 @@ interface UrlParam {
 interface App {
   pathname?: string;
   hash?: string;
+  search?: string;
 }
```

Comment:
I don't know where this util came from, or why the operations team is responsible for maintaining it, but it feels unnecessarily limiting to restrict the properties that `App` can override from `UrlParam`... Why not just type both arguments to `UrlParam`?

Reply:
Done